# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../qmail-autoresponder/remove-mysql.diff
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -Naur qmail-autoresponder-0.97.orig/options.c qmail-autoresponder-0.97/options.c
--- qmail-autoresponder-0.97.orig/options.c	2009-09-26 22:45:17.000000000 +0200
+++ qmail-autoresponder-0.97/options.c	2009-09-26 22:46:07.000000000 +0200
@@ -1,4 +1,3 @@
-#include <mysql/mysql.h>
 #include <stdlib.h>
 #include <string.h>
 #include <str/str.h>
